sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -273,13 +735,15 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [formreco_nuget_package]: https://www.nuget.org/packages/Azure.AI.FormRecognizer
 [formreco_samples]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md
 [formrecov3_samples]: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/V3.1/README.md
+[formreco_changelog]: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md
 [formreco_rest_api]: https://westus2.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-1/operations/AnalyzeBusinessCardAsync
```

Comment:
Do we have a new REST API link?

Reply:
haven't seen one yet and I'm guessing it won't be published until next week. :( might want to aka.ms this one

Reply:
I don't think so... maybe we can remove this since the error document provided by the service team will have that information?

Reply:
Updated link to https://aka.ms/azsdk/formrecognizer/restapi (currently pointing to V2).